src/data_loading.py

```diff
@@ -332,36 +335,82 @@ def get_api_blood_data(api_root = 'https://api.a2cps.org/files/v2/download/publi
         traceback.print_exc()
         return None
 
-def get_api_subjects_json(api_root = 'https://api.a2cps.org/files/v2/download/public/system/a2cps.storage.community/reports'):
+def get_api_subjects_json(api_root = 'https://api.a2cps.org/files/v2/download/public/system/a2cps.storage.community/reports', 
+                          vbr_api_root = os.environ.get('VBR_API_ROOT'), 
+                          portal_api_root = os.environ.get('PORTAL_API_ROOT'), 
+                          coresessionid = None):
     ''' Load subjects data from api. Note data needs to be cleaned, etc. to create properly formatted data product'''
+    print(coresessionid[:5])
+    tapis_token = get_tapis_token(portal_api_root, coresessionid)
+    print(tapis_token)
+    auth_status = get_auth_status(vbr_api_root, tapis_token)
+
+    if auth_status == True:
+        try:
+            # Load Json Data
+            subjects1_filepath = '/'.join([api_root,'subjects','subjects-1-latest.json'])
+            subjects1_request = requests.get(subjects1_filepath)
```

Comment:
I would have suspected that this call and the call on line 360 would require an auth (i.e. the bearer token). or am I mistaken there?  I know this is the last step of getting things to work so please ignore it till when appropriate.